Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -5998,5 +5998,37 @@ IPostprocessComposer<T> ConfigurePropertyField<T>(IPostprocessComposer<T> compos
                 return composer.With(x => x.Value, "42");
             }
         }
+
+        private class TypeWithRangedTimeSpanProperty
+        {
+            [Range(typeof(TimeSpan), "00:00:00", "12:00:00")]
```

Comment:
Could you please start range from e.g. `02:00:00`? I simply afraid that minimum value looks too default, so if code doesn't work, test will still pass 😅